### PR TITLE
Upgrade to Google Analytics 4 api

### PR DESCRIPTION
**Pull request by benhutton**

### CODE REVIEW

File: wakes.gemspec [hunk 1]
```diff
@@ -26,7 +26,8 @@ Gem::Specification.new do |spec|
   spec.require_paths = ['lib']
   spec.test_files = Dir['spec/**/*']
 
-  spec.add_dependency 'google-api-client'
+  spec.add_dependency 'google-apis-analyticsdata_v1beta', '~> 0.1'
```
Comment (gaganawhad) [inline wakes.gemspec:29]: @benhutton Interesting. So they are still on a beta version!

File: spec/wrappers/wakes/google_analytics_api_wrapper_spec.rb [hunk 2]
```diff
@@ -4,10 +4,11 @@
 
 RSpec.describe Wakes::GoogleAnalyticsApiWrapper do
   describe '#get_page_of_pageviews' do
-    let(:rows) { [['/path', 151], ['/path/2', 500]] }
-    let(:results) { double('Results', :rows => rows, :items_per_page => 2) }
+    let(:row1) { double('Row', dimension_values: [double('DimensionValue', value: '/path')], metric_values: [double('MetricValue', value: 151)]) }
```
Comment (gaganawhad) [inline spec/wrappers/wakes/google_analytics_api_wrapper_spec.rb:7]: @benhutton Can we use dummy paths like `/resources/message-1` and `/resources/article-1` as the dummy values here? That will help a reader get better oriented.

**Review by gaganawhad — APPROVED** [review]

File: wakes.gemspec (hunk 1 above)
Comment (benhutton) [reply wakes.gemspec:29]: yeah, this is all pretty new!

File: spec/wrappers/wakes/google_analytics_api_wrapper_spec.rb (hunk 2 above)
Comment (benhutton) [reply spec/wrappers/wakes/google_analytics_api_wrapper_spec.rb]: done!

**Pull request merged**